### PR TITLE
Make the use of braces consistent with project C++ guidelines

### PR DESCRIPTION
**Pull request by bihanssen**
The project's C++ guidelines state to always put opening and closing braces on a separate line.

### CODE REVIEW

File: src/driver.cpp [hunk 1]
```diff
@@ -1379,6 +1382,7 @@ void Adapter::AfterGetBleOption(uv_work_t *req)
     }
 
     baton->callback->Call(2, argv);
+    delete baton->p_opt;
```
Comment (mrodem) [inline src/driver.cpp]: Just to be sure: This line is not related to brace formatting. Should perhaps have been on a separate commit?

Comment (bihanssen) [inline src/driver.cpp]: Look at that, it got unintentionally squashed in.

Comment (bihanssen) [inline src/driver.cpp]: Fixed in commit 465d87f
